pages/overview/what-is-jsonschema.md

```diff
@@ -2,219 +2,56 @@
-* **Highly extensible** and can be tailored to fit your needs.
-    * You can create *custom keywords*, *formats*, and *validation rules* to suit your own requirements.
-* **Validate your data**, which helps you:
-    * **Automate testing**: JSON Schema validation enables automated testing, ensuring that data consistently adheres to the specified rules and constraints.
-    * **Enhance data quality**: By enforcing validation rules, JSON Schema helps ensure the quality of client-submitted data, preventing inconsistencies, errors, and malicious inputs.
-* **Wide range of tools availability**: The JSON Schema community has a wealth of tools and resources available across many programming languages to help you create, validate, and integrate your schemas.
-
-
-## Why organizations adopt JSON Schema
-
-* **Streamline testing and validation**: Simplify your validation logic to reduce your code’s complexity and save time on development. Define constraints for your data structures to catch and prevent errors, inconsistencies, and invalid data.
-* **Exchange data seamlessly**: Establish a common language for data exchange, no matter the scale or complexity of your project. Define precise validation rules for your data structures to create shared understanding and increase interoperability across different systems and platforms.
-* **Document your data**: Create a clear, standardized representation of your data to improve understanding and collaboration among developers, stakeholders, and collaborators.
-* **Vibrant tooling ecosystem**: Adopt JSON Schema with an expansive range of community-driven tools, libraries, and frameworks across many programming languages.
 
-## JSON Schema History
+## How does JSON Schema work?
 
-JSON Schema has a rich history that dates back to the [first JSON Schema proposal](https://web.archive.org/web/20071026185150/http://json.com/json-schema-proposal/) submitted by **Kris Zyp** to json.com on October 2nd, 2007. 
+In the world of data exchange, JSON Schema emerges as a powerful tool for defining the structure and rules of your JSON data. It uses keywords to define the properties of your data. These keywords allow you to specify details, including datatypes, required properties that must be present for a valid JSON document, and optional properties that can be included. 
 
-The current version of JSON Schema is [2020-12](../draft/2020-12/release-notes), which represents the latest advancements and have expended capabilities as compared with the previous version `draft-04`, `draft-06`, `draft-07`. We encourage everyone to adopt the latest version whenever possible to take advantage of all the advancements and benefits of JSON Schema.
+Furthermore, you can restrict the length of your data and ensure it adheres to specific formats. JSON Schema empowers you to define minimum and maximum values for numerical data, keeping your numbers within a valid range.
```

Comment:
For this paragraph, instead of focused in the particularities of properties and options, I'd say this paragraph should be to highlight the value of JSON Schema. This paragraph may be starting with something like "By adopting JSON Schema organizations can seamlessly and consistently manage and exchange data" ... or any other alternative playing with the messages of benefits of json schema detailed later in the doc.

Reply:
@benjagm The benefits to the organization are already outlined below, so there is no need to add "by adopting JSON Schema, organizations can seamlessly and consistently manage and exchange data". It does not correspond with the headline